dev_tools/nbformat

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Formats ipython notebooks with tensorflow-docs nbformat tool.
+#
+# Usage:
+#     dev_tools/nbformat [--apply]
+#
+# Without '--apply', the diff that would be applied is printed and the exit
+# status is 1 if there are any changes or else 0 if no changes are needed.
+#
+# With '--apply', the exit status is 0 and the changed files are actually
+# reformatted.
+#
+################################################################################
+
+only_print=1
+for arg in $@; do
+    if [[ "${arg}" == "--apply" ]]; then
+        only_print=0
+    else
+        echo -e "\033[31mToo many arguments. Expected [--apply].\033[0m" >&2
+        exit 1
+    fi
+done
+
+# Get the working directory to the repo root.
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$(git rev-parse --show-toplevel)"
+
+# Install a pinned version of tf-docs
+TF_DOCS_COMMIT=b3dc8a922d8bdc6e998a8ad4f4953359dc6e576a
```

Comment:
Pinning a commit makes sense, but we don't really do releases in tensorflow/docs so this commit is arbitrary.

It might be better to download the cirq `dev_tools/nbformat` script into your GitHub Actions VM so you're at least using the same thing and don't get too out of sync.

Reply:
+1 downloading cirq's script